quadlet/minio-prometheus/prometheus.network

```diff
@@ -0,0 +1,3 @@
+[Network]
+NetworkName=prometheus
+Internal=true
```

Comment:
I guess the `Internal` flag is part of the reason why you attach two networks. Can you please elaborate on what you're trying to achieve? I'm not sure there isn't a simpler way to achieve it.